src/components/counter/counter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './counter.css';
+class Counter extends React.Component {
```

Comment:
Please leave line space between imports and class.

Reply:
Done also formatted.